alerts/apps_alerts.libsonnet

```diff
@@ -143,7 +143,7 @@
           {
             expr: |||
               (
-                max without (revision) (
```

Comment:
Pint suggests a refactor here.

```
┌[rexagod@nebuchadnezzar] [/dev/ttys000] [pint ⚡] [2]
└[~/repositories/oss/kubernetes-mixin]> make pint-lint 

prometheus_alerts.yaml:106-123 Warning: Aggregation using `without()` can be fragile when used inside binary expression because both sides must have identical sets of labels to produce any results, adding or removing labels to metrics used here can easily break the query, consider aggregating using `by()` to ensure consistent labels. (promql/fragile)
 106 |     "expr": |
 107 |       (
 108 |         max without (revision) (
 109 |           kube_statefulset_status_current_revision{job="kube-state-metrics"}
 110 |             unless
 111 |           kube_statefulset_status_update_revision{job="kube-state-metrics"}
 112 |         )
 113 |           *
 114 |         (
 115 |           kube_statefulset_replicas{job="kube-state-metrics"}
 116 |             !=
 117 |           kube_statefulset_status_replicas_updated{job="kube-state-metrics"}
 118 |         )
 119 |       )  and (
 120 |         changes(kube_statefulset_status_replicas_updated{job="kube-state-metrics"}[5m])
 121 |           ==
 122 |         0
 123 |       )
make: *** [pint-lint] Error 1
```